build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java'
+    id 'java-library'
```

Comment:
do you need both 'java' and 'java-library'?

Reply:
Maybe not?